checks.d/openvpn.py

```diff
@@ -0,0 +1,42 @@
+import csv
+from collections import defaultdict
+
+# project
+from checks import AgentCheck
+
+
+# Each user really should only have one active connection
```

Comment:
Rather than encode this here I think we'd rather have this in a monitor, so we don't have to puppet everything if we change it!